docs/theme/mlflow/static/css/tabs.css

```diff
@@ -95,6 +95,82 @@ border-bottom: 2px solid rgb(50, 50, 50);
 background-color: rgb(50, 50, 50);
 }
 
+/*
+ * Main Landing page Tab display panes
+ */
+
+ .main-container {
+    margin: 1rem;
+    text-align: left;
+    min-height: 500px;
+}
+
+.main-container h3 {
+    margin-bottom: 30px;
+}
+
+.sub-container-two-columns {
```

Comment:
this is non-blocking, but it'd be nice if the image could wrap below the text if the screen is small. at the moment, it looks like this:

<img width="431" alt="Screenshot 2024-05-16 at 1 09 29 PM" src="https://github.com/mlflow/mlflow/assets/148037680/339ee105-2e38-4053-a2dd-b4e9494cc1d6">

this can be done with a media query that replaces the two-column grid with just a normal flex layout, like:

```
@media (max-width: 600px) {
    .sub-container-two-columns {
        display: flex;
        flex-direction: column;
    }
}
```

Reply:
Great catch. Added!